glib-sys/src/lib.rs

```diff
@@ -3658,37 +3677,37 @@ extern "C" {
     pub fn g_file_error_from_errno(err_no: c_int) -> GFileError;
     pub fn g_file_error_quark() -> GQuark;
     #[cfg(any(windows, feature = "dox"))]
-    pub fn g_file_get_contents_utf8(filename: *mut c_char, contents: *mut *mut u8, length: *mut size_t, error: *mut *mut GError) -> gboolean;
-    pub fn g_file_get_contents(filename: *mut c_char, contents: *mut *mut u8, length: *mut size_t, error: *mut *mut GError) -> gboolean;
+    pub fn g_file_get_contents_utf8(filename: *const c_char, contents: *mut *mut u8, length: *mut size_t, error: *mut *mut GError) -> gboolean;
+    pub fn g_file_get_contents(filename: *const c_char, contents: *mut *mut u8, length: *mut size_t, error: *mut *mut GError) -> gboolean;
     #[cfg(any(windows, feature = "dox"))]
-    pub fn g_file_open_tmp_utf8(tmpl: *mut c_char, name_used: *mut *mut c_char, error: *mut *mut GError) -> c_int;
-    pub fn g_file_open_tmp(tmpl: *mut c_char, name_used: *mut *mut c_char, error: *mut *mut GError) -> c_int;
-    pub fn g_file_read_link(filename: *mut c_char, error: *mut *mut GError) -> *mut c_char;
-    pub fn g_file_set_contents(filename: *mut c_char, contents: *mut u8, length: ssize_t, error: *mut *mut GError) -> gboolean;
+    pub fn g_file_open_tmp_utf8(tmpl: *const c_char, name_used: *mut *mut c_char, error: *mut *mut GError) -> c_int;
+    pub fn g_file_open_tmp(tmpl: *const c_char, name_used: *mut *mut c_char, error: *mut *mut GError) -> c_int;
+    pub fn g_file_read_link(filename: *const c_char, error: *mut *mut GError) -> *mut c_char;
+    pub fn g_file_set_contents(filename: *const c_char, contents: *mut u8, length: ssize_t, error: *mut *mut GError) -> gboolean;
     #[cfg(any(windows, feature = "dox"))]
-    pub fn g_file_test_utf8(filename: *mut c_char, test: GFileTest) -> gboolean;
-    pub fn g_file_test(filename: *mut c_char, test: GFileTest) -> gboolean;
-    pub fn g_filename_display_basename(filename: *mut c_char) -> *mut c_char;
-    pub fn g_filename_display_name(filename: *mut c_char) -> *mut c_char;
+    pub fn g_file_test_utf8(filename: *const c_char, test: GFileTest) -> gboolean;
```

Comment:
Why are these all changing?

Reply:
Because it has const now: `<type name="filename" c:type="const gchar*"/>`